code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -69,6 +69,27 @@ namespace <#= qualifiedNamespace #>
                 entityManager.AddComponentData(entity, data);
                 entityManager.AddComponentData(entity, new NotAuthoritative<<#= componentDetails.TypeName #>>());
 
+                var update = new <#= componentDetails.TypeName #>.Update 
+                {
+<# foreach (var fieldDetails in fieldDetailsList) { #>
+                    <#= fieldDetails.PascalCaseName #> = data.<#= fieldDetails.PascalCaseName #>,
+<# } #>
+                };
+                
+                var updates = new List<<#= componentDetails.TypeName #>.Update>
+                {
+                    update
+                };
+                
+                var updatesComponent = new <#= componentDetails.TypeName #>.ReceivedUpdates
+                {
+                    handle = ReferenceTypeProviders.UpdatesProvider.Allocate(World)
+                };
+                
+                ReferenceTypeProviders.UpdatesProvider.Set(updatesComponent.handle, updates);
+                updatesComponent.Updates.Add(update);
```

Comment:
Don't need this as the list is initialised with the update

Reply:
:/ Done :)